client/src/components/subscription/updateSubscription.js

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
 import { toast } from 'react-toastify';
+import ReactDayPicker from '../utils/datepicker';
+
+const WEEKLY_DAYS = {
+  0: '2022-01-02T00:00:00',
+  1: '2022-01-03T00:00:00',
+  2: '2022-01-04T00:00:00',
+  3: '2022-01-05T00:00:00',
+  4: '2022-01-06T00:00:00',
+  5: '2022-01-07T00:00:00',
+  6: '2022-01-08T00:00:00',
+};
```

Comment:
this is fine for now, but obviously in 2023 we'd want to update this. perhaps in the future we can update this to be dynamic, but i think it's fine for now